ckanext/dia_theme/templates/snippets/topbar_links.html

```diff
@@ -1,2 +0,0 @@
-<li class="feedback"><a href="{{ h.parent_site_url() }}/request-data">Request dataset</a></li>
-<li><a href="{{ h.parent_site_url() }}/add-dataset">Add dataset</a></li>
```

Comment:
We'll need to check the header heights once we've done this on CWP. We can use UAT to test.

Reply:
Did you revise the header height after changing padding on the CWP site @matt-in-a-hat ?